### PR TITLE
OpenTelemetry PHP Beta Blog Post

### PR DESCRIPTION
**Pull request by bobstrecansky**
Creating an OpenTelemetry PHP Beta Announcement Blog Post

### CODE REVIEW

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: {{ Opentelemetry-php Beta Release }}
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:2]:
```suggestion
title: OpenTelemetry PHP Beta Release
```

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+linkTitle: PHP Beta Release # Mandatory, make sure that your short title.
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:3]:
```suggestion
linkTitle: PHP Beta Release
```

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: {{ Opentelemetry-php Beta Release }}
+linkTitle: PHP Beta Release # Mandatory, make sure that your short title. 
+date: 2023-01-25
+author: " [Bob Strecansky](https://github.com/bobstrecansky/) (Intuit)"
+draft: true
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:6]:
```suggestion
```

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+OpenTelemetry PHP Beta Release Blog Post
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:8]:
```suggestion
```

The title will be set via the `title` above.

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+The OpenTelemetry PHP SIG is very excited to announce the release of [v1.0.0beta1](https://github.com/open-telemetry/opentelemetry-php/releases/tag/1.0.0beta1) of OpenTelemetry PHP.  This is the culmination of over 3 years of work done by the OpenTelemetry PHP team.  
+
+We are actively soliciting feedback from the development community for this library.  Please check out the repo, instrument your PHP app with the product, and open an issue if you’d like to see a bug squashed or a new feature added.
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:12]:
```suggestion
We are actively soliciting feedback from the development community for this library. Please check out the repo, instrument your PHP app with the product, and open an issue if you’d like to see a bug squashed or a new feature added.
```

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+We have lots of [examples](https://github.com/open-telemetry/opentelemetry-php/tree/main/examples) in the repo to get you started, or you can check out the [quote service](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/quoteservice.md), a demo application built in PHP to showcase the library.
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:14]:
```suggestion
We have lots of [examples](/docs/instrumentation/php/examples/) in the repo to get you started, or you can check out the [quote service](https://github.com/open-telemetry/opentelemetry-demo/blob/main/docs/services/quoteservice.md), a demo application built in PHP to showcase the library.
```

Pointing to the examples link here in the doc. Also do you want to call out the Getting Started in the docs here?

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+Questions?  Feel free to reach out to us in the CNCF [#otel-php](https://cloud-native.slack.com/archives/C01NFPCV44V) slack channel, or come to our SIG meeting (which you can find on the [otel public calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com))
```
Comment (svrnm) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:16]:
```suggestion
Questions? Feel free to reach out to us in the CNCF [#otel-php](https://cloud-native.slack.com/archives/C01NFPCV44V) slack channel, or come to our SIG meeting, which you can find on the [otel public calendar](https://calendar.google.com/calendar/embed?src=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com)
```

**Review by svrnm — CHANGES_REQUESTED** [review]
Exciting :-)

**Comment by svrnm** [comment]
@chalin @cartermp since the banner announcing the APAC End User Group is up there for a while now, maybe we want to replace it with an PHP Beta announcement?

**Comment by cartermp** [comment]
Yes, absolutely. This is a real banger IMO, deserves a banner!

File: content/en/blog/2023/opentelemetry-php-beta-release/index.md (hunk 5 above)
Comment (cartermp) [inline content/en/blog/2023/opentelemetry-php-beta-release/index.md:12]: Link to repo would be great

**Comment by chalin** [comment]
Was hoping to see a preview, but I guess that the PR isn't building yet... ?

**Comment by chalin** [comment]
I'll make a pass once the PR builds.

**Comment by bobstrecansky** [comment]
@svrnm - I think I addressed all your changes.  Please let me know if there are additional things you'd like me to change.

**Review by cartermp — APPROVED** [review]
Thanks! Excited to see this go 🚀

**Review by chalin — CHANGES_REQUESTED** [review]
Rename:

- From `content/en/blog/2023/opentelemetry-php-beta-release/index.md`
- To `content/en/blog/2023/php-beta-release.md`

There's something wrong with this PR because it doesn't build.

@bobstrecansky - I'd suggest closing and reopening it again after you pull the latest changes and rename the file as suggested above.

**Review by chalin — APPROVED** [review]
Actually, it builds for me locally, so I'm ok with merging this once @svrnm and @cartermp are happy. (Btw, I made a few copyedits.)

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
Ah... @bobstrecansky:


> [bobstrecansky](https://github.com/bobstrecansky) wants to merge 7 commits into [open-telemetry:opentelemetry_php_beta_blog_post](https://github.com/open-telemetry/opentelemetry.io/tree/opentelemetry_php_beta_blog_post) from [bobstrecansky:opentelemetry_php_beta_blog_post](https://github.com/bobstrecansky/opentelemetry.io/tree/opentelemetry_php_beta_blog_post)

Can you raise the PR against main? TY

**Comment by bobstrecansky** [comment]
Sorry about that, what a bonehead mistake 🤦🏼 
https://github.com/open-telemetry/opentelemetry.io/pull/2219

**Pull request closed without merging**